libs/go/athenzconf/Makefile

```diff
@@ -10,18 +10,19 @@ GOPKGNAME = github.com/AthenZ/athenz/libs/go/athenzconf
 
 # check to see if go utility is installed
 GO := $(shell command -v go 2> /dev/null)
-export GOPATH=$(PWD)
+GOPATH := $(shell pwd)
```

Comment:
Noticed that in the clients Makefiles, the GOPATH was changed from local dir (PWD) to (/tmp/go):
export GOPATH ?= /tmp/go
Not sure why the directory was changed there but not here